### DISCUSSION TITLE
[data] Handle different cases of vertical text

### DISCUSSION THREAD

**Discussion by charlesmindee** [mindee/doctr#406] [Ideas]
We need to keep in mind that we will come across 2 cases of vertical text:
- "Rotated" vertical text: horizontal text with a +/-90° rotation (rotated letters)

![rotated](https://user-images.githubusercontent.com/70526046/110812310-ca3b1e80-8287-11eb-8605-17df2e8dff47.jpg)

- "Truly" vertical text: a text with horizontal letters (unrotated letters), written from top to bottom

![vertical](https://user-images.githubusercontent.com/70526046/110813122-7f6dd680-8288-11eb-98db-de03dc224201.jpg)

**Comment by charlesmindee** [comment]
Moving since to a discussion, since it is a long term and very opened issue.

**Reply by mllife** [reply]
@fg-mindee @charlesmindee, any update in handling vertical or rotated text? 
<img width="754" alt="Screenshot 2024-03-26 at 11 23 08 AM" src="https://github.com/mindee/doctr/assets/69006366/9c168815-0246-47d4-b4e2-0828fcaa480f">
I am unable to find text inside this image.
